testsuite/src/it/java/org/graalvm/tests/integration/utils/Commands.java

```diff
@@ -109,6 +127,23 @@ public static String getProperty(String[] alternatives, String defaultValue) {
         return prop;
     }
 
+    public static boolean isVersion22_3OrBetter(String version) {
+        if (version == null) {
+            return false;
+        }
+        try {
+            String[] tokens = version.split("\\.");
+            if (tokens.length < 2) {
+                return false;
+            }
+            int major = Integer.parseInt(tokens[0]);
+            int minor = Integer.parseInt(tokens[1]);
+            return major >= 22 && minor > 2;
+        } catch (NumberFormatException e) {
+            return false; // assume default of 21
+        }
```

Comment:
I think you can simplify this with:

```suggestion
        return Version.parse(version).compareTo(22, 3) >= 0;
```

Reply:
Thanks, the latest version uses that now.